.changelog/3990.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed an issue where custom content items with special YAML characters (like `:`) in their names would cause an error during UUID ID replacement.
```

Comment:
```suggestion
- description: Fixed an issue where downloading custom content items with special YAML characters (like `:`) in their names would cause an error during UUID ID replacement when using **download** command.
```